lib/sycamore/sycamore/llms/openai.py

```diff
@@ -399,13 +409,18 @@ def generate(self, *, prompt_kwargs: dict, llm_kwargs: Optional[dict] = None) ->
         self._cache_set(key, value)
         return ret
 
-    def _generate_using_openai(self, prompt_kwargs, llm_kwargs) -> str:
+    def _generate_using_openai(self, prompt_kwargs, llm_kwargs) -> Any:
```

Comment:
Can we have a separate method for the openai structured responses? We had a bug earlier because this function had an Any return type so it's safer to keep them separate

Reply:
fixed